package.json

```diff
@@ -65,6 +65,6 @@
     "redux-devtools-extension": "^2.13.2",
     "tslint": "^5.9.1",
     "tslint-config-prettier": "^1.10.0",
-    "typescript": "^2.8.1"
+    "typescript": "^3.0.0-dev.20180609"
```

Comment:
why dev version?

Reply:
There’s a bugfix in the latest Typescript stuff that fixes how nested imports work, can’t point to the specific PR but i know 2.9.1 fails but 3.0.0-dev works. That said until testing works properly Idk if we should merge this. May be able to fix that with precompilation in the other packages but i’m not 100% sure how that’d work, never done it before